kaldb/src/main/java/com/slack/kaldb/server/KaldbIndexer.java

```diff
@@ -79,7 +78,8 @@ public KaldbIndexer(
   protected void startUp() throws Exception {
     LOG.info("Starting Kaldb indexer.");
     long startOffset = indexerPreStart();
-    chunkManager.awaitRunning(DEFAULT_START_STOP_DURATION);
+    // ensure the chunk manager is available to receive messages
+    chunkManager.awaitRunning();
```

Comment:
Just confirming that guava service manager timeout would be applied now in case the service fails to start.

Reply:
This would be expected to wait indefinitely on the `chunkManager` before the indexer will start indexing (depending on healthcheck configs). Having a timeout here doesn't make a ton of sense, because if the chunk manager isn't ready there's no sense in proceeding reading messages from Kafka.

Reply:
Since a timeout would terminate the process isn't that preferable than a hung process?